checkov/terraform/checks/resource/alicloud/LaunchTemplateDisksAreEncrypted.py

```diff
@@ -0,0 +1,22 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceCheck
+
+
+class LaunchTemplateDisksAreEncrypted(BaseResourceCheck):
```

Comment:
@JamesWoolfenden `BaseResourceValueCheck`?

Reply:
Should be baseresourcecheck. fixed